README.en.md

```diff
@@ -0,0 +1,179 @@
+# Command Line Tool Introduction
+Welcome to Tencent Cloud Command Line Tool (TCCLI), a unified tool for managing Tencent cloud resources. With Tencent Cloud Command Line Tools, you can quickly and easily call the Tencent Cloud API to manage your Tencent cloud resources. You can also automate and script based on Tencent Cloud's command-line tools, which can be combined and reused in a variety of ways.
+# Install TCCLI
+1. Install the Python environment and Pip tools. Before installing the command line tools, make sure your system has the Python environment and Pip tools installed. **Note that the python version must be 2.7 and above**, please refer to [python homepage](https://www.python.org/) and [pip homepage](https://pypi.org/project/pip/).
+2. TCCLI relies on the TencentCloudApi Python SDK. **If the version number of the TencentCloudApi Python SDK is smaller than the TCCLI version number to be installed, the TencentCloudApi Python SDK** will be automatically upgraded when TCCLI is installed.
+3. Install TCCLI and execute the following command:
+```bash
+Pip install tccli
+```
+4. After the installation is complete, execute tccli version to check whether the installation is successful.
+5. If your environment is a Linux environment, you can enable auto-completion with the following command:
+```bash
+Complete -C 'tccli_completer' tccli
+```
+
+# Configure TCCLI
+To use the Tencent Cloud Command Line tool, you also need to do some initial configuration to make it necessary to use the Cloud API.
+1. Interactive mode, you can use the tccli configure command to enter interactive mode for quick configuration.
+
+```bash
+$ tccli configure
+TencentCloud API secretId [*afcQ]: AKIDwLw1234MMfPRle2g9nR2OTI787aBCDP
+TencentCloud API secretKey [*ArFd]:OxXj7khcV1234dQSSYNABcdCc1LiArFd
+Region: ap-guangzhou
+Output[json]:
+```
+secretId: Cloud API key SecretId.
+secretIKey: Cloud API key SecretKey.
+Region: Cloud product area, please go to the corresponding product page to get the available region.
+Output: Optional parameter, request return packet output format, support [json table text] three formats, the default is json.
+For more information, please see tccli configure help.
+
+2. Command line mode, you can configure your information in an automation script via command line mode.
+```bash
+The #set subcommand can set a certain configuration or configure multiple at the same time.
+Tccli configure set secretId AKIDwLw1234MMfPRle2g9nR2OTI787aBCDP
```

Comment:
When it is a command, you should not capitalize it, always be smaller case: Tccli -> tccli
The following commands all have same problem, please fix them, thanks